[DO NOT MERGE] EUI manual testing

mgadewoll opened the pull request:
>[!important]
DO NOT MERGE THIS PR.
This PR is only for manual testing of EUI features.

elasticmachine commented:
<!-- buildkite-pr-bot-trigger-comment -->:robot: Jobs for this PR can be triggered through checkboxes. :construction: 

:information_source: To trigger the CI, please tick the checkbox below :point_down: 

- [ ] <!-- buildkite-pr-bot-checkbox-trigger:kibana-pull-request --> Click to trigger **kibana-pull-request** for this PR!
- [ ] <!-- buildkite-pr-bot-checkbox-trigger:kibana-deploy-project-from-pr --> Click to trigger **kibana-deploy-project-from-pr** for this PR!
- [ ] <!-- buildkite-pr-bot-checkbox-trigger:kibana-deploy-cloud-from-pr --> Click to trigger **kibana-deploy-cloud-from-pr** for this PR!

kibanamachine committed:
Changes from node scripts/lint_ts_projects --fix

kibanamachine committed:
Changes from node scripts/regenerate_moon_projects.js --update

mgadewoll committed:
test: update tests and snapshots

mgadewoll committed:
refactor: adjust TagSearchBarOption to follow new EuiSelectableListItem styles

mgadewoll committed:
test: update EuiComboBox tests

mgadewoll committed:
refactor: remove deprecated props size and gutterSize

mgadewoll committed:
test: update tests and snapshots

mgadewoll committed:
chore: update i18n

mgadewoll committed:
refactor: remove flush prop from EuiListGroup usages

mgadewoll committed:
refactor: remove custom styles from FilterTypeFilter

mgadewoll committed:
refactor: remove size prop on EuiContextMenu, EuiContextMenuPanel and EuiContextMenuItem

mgadewoll committed:
test: update tests and snapshots

- due to changed context menu item role

mgadewoll committed:
refactor: remove hasDividers prop on EuiSuperSelect

mgadewoll committed:
test: update tests and snapshots

mgadewoll committed:
chore: update eui package

- contains UI changes to EuiSelectable, EuiComboBox, EuiListGroup, EuiContextMenu and EuiSuperSelect

mgadewoll committed:
refactor: update EuiSelectable usages

- adjusts padding usages; paddingSize now adds a padding on the list container not the list item; where possible we use the parent container padding, otherwise the list padding

mgadewoll committed:
refactor: cleanup more EuiListGroup size props

mgadewoll committed:
refactor: cleanup more EuiContextMenu size props

mgadewoll committed:
refactor: remove flush and size prop on EuiListGroup

mgadewoll committed:
test: update snapshots

mgadewoll committed:
refactor: component usage cleanups to align expected output

mgadewoll committed:
chore: update eui package

mgadewoll committed:
refactor: remove more hasDividers on EuiSuperSelect

mgadewoll committed:
test: update snapshots

mgadewoll committed:
chore: update eui package

- includes all latest changes on selection list components

mgadewoll committed:
refactor: remove itemLayoutAlign prop on EuiSuperSelect

mgadewoll committed:
test: update snapshots

mgadewoll committed:
Merge branch 'main' into eui/selection-list-ui-updates

mgadewoll committed:
refactor: remove size prop after merging main

mgadewoll committed:
Merge branch 'main' into eui/selection-list-ui-updates

mgadewoll committed:
refactor: more usage adjustments

mgadewoll committed:
Merge branch 'main' into eui/selection-list-ui-updates

mgadewoll committed:
test: update test

ensures combobox panel is closed to enable clicking a checkbox

mgadewoll committed:
chore: update eui package

mgadewoll committed:
Merge remote-tracking branch 'upstream' into eui/selection-list-ui-updates

mgadewoll committed:
test: update tests

- adjust to use VariableSizeList to align with internal change in EuiSelectable and EuiComboBox

mgadewoll committed:
fix: remove custom checkbox output from policy select

mgadewoll committed:
chore: updte eui package

mgadewoll committed:
test: update policy_selector tests

kibanamachine commented:
## :yellow_heart: Build succeeded, but was flaky
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/436121)
* Commit: 1f05bb252104606cd608d35fa366b2d78a7c070d
* [Storybooks Preview](https://ci-artifacts.kibana.dev/storybooks/pr-244898)

### Failed CI Steps
* [Jest Tests #1](https://buildkite.com/elastic/kibana-pull-request/builds/436121#019ddd94-c3b2-4ab7-90c7-bd8932281ced)
* [FTR Configs #148](https://buildkite.com/elastic/kibana-pull-request/builds/436121#019ddd94-c3f1-4f8a-81e4-a685a4d1859d)
* [Agent Builder API Smoke Tests #1](https://buildkite.com/elastic/kibana-pull-request/builds/436121#019ddd91-9dd2-426f-a221-950b41655dbe)

### Test Failures
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/436121#019ddd94-c3b2-4ab7-90c7-bd8932281ced) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/436121/jobs/019ddd94-c3b2-4ab7-90c7-bd8932281ced/artifacts/019dddb9-9686-414a-8bfb-af1dcb6dacb4) Jest Tests #<span></span>1 / <HighlighedFieldsModal /> should update rule when save button is clicked
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/436121#019ddd91-9dd2-426f-a221-950b41655dbe) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/436121/jobs/019ddd91-9dd2-426f-a221-950b41655dbe/artifacts/019ddda8-cc76-4d23-a067-894b1eb942cc) Agent Builder API Smoke Tests #<span></span>1 / Agent Builder - LLM Smoke tests EIS Models (dynamically configured) Connector: openai-gpt-5.4-pro returns an answer for a simple message

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
#### Module Count
> Fewer modules leads to a faster build time


| id | [before](https://github.com/elastic/kibana/commit/9a0ff9aa5714a7f550f8abf4b3a40f8d3c9ccf19) | [after](https://github.com/elastic/kibana/commit/1f05bb252104606cd608d35fa366b2d78a7c070d) | diff |
| --- | --- | --- | --- |
| `maps` | 1374 | 1375 | +1 |

#### Any counts in public APIs
> Total count of every any typed public API. Target amount is 0. Run `node scripts/build_api_docs --plugin [yourplugin] --stats any` for more detailed information.


| id | [before](https://github.com/elastic/kibana/commit/9a0ff9aa5714a7f550f8abf4b3a40f8d3c9ccf19) | [after](https://github.com/elastic/kibana/commit/1f05bb252104606cd608d35fa366b2d78a7c070d) | diff |
| --- | --- | --- | --- |
| `@kbn/alerts-ui-shared` | 0 | 2 | +2 |
| `@kbn/cloud-security-posture` | 0 | 2 | +2 |
| `@kbn/content-list-table` | 10 | 11 | +1 |
| `@kbn/content-management-favorites-public` | 0 | 1 | +1 |
| `@kbn/core-chrome-app-menu-components` | 0 | 1 | +1 |
| `@kbn/discover-utils` | 0 | 1 | +1 |
| `@kbn/grok-ui` | 0 | 1 | +1 |
| `@kbn/index-management-shared-types` | 3 | 4 | +1 |
| `@kbn/ml-anomaly-utils` | 3 | 6 | +3 |
| `@kbn/react-kibana-context-common` | 0 | 1 | +1 |
| `triggersActionsUi` | 1 | 3 | +2 |
| total |  |  | +16 |

#### Async chunks
> Total size of all lazy-loaded chunks that will be downloaded as the user navigates the app


| id | [before](https://github.com/elastic/kibana/commit/9a0ff9aa5714a7f550f8abf4b3a40f8d3c9ccf19) | [after](https://github.com/elastic/kibana/commit/1f05bb252104606cd608d35fa366b2d78a7c070d) | diff |
| --- | --- | --- | --- |
| `agentBuilder` | 1.1MB | 1.1MB | -244.0B |
| `alertingVTwo` | 679.7KB | 679.5KB | -162.0B |
| `apm` | 2.8MB | 2.8MB | -171.0B |
| `cases` | 2.3MB | 2.3MB | +5.0B |
| `cloudDefend` | 93.7KB | 93.7KB | -36.0B |
| `cloudSecurityPosture` | 613.5KB | 613.4KB | -155.0B |
| `console` | 207.1KB | 207.1KB | -9.0B |
| `contentConnectors` | 475.4KB | 475.3KB | -27.0B |
| `controls` | 522.9KB | 525.5KB | +2.7KB |
| `dashboard` | 1.1MB | 1.1MB | +2.6KB |
| `datasetQuality` | 524.5KB | 524.5KB | +85.0B |
| `dataUsage` | 28.0KB | 28.0KB | -24.0B |
| `dataViewManagement` | 144.7KB | 144.8KB | +42.0B |
| `dataVisualizer` | 618.2KB | 618.2KB | -46.0B |
| `discover` | 1.9MB | 1.9MB | -344.0B |
| `embeddableAlertsTable` | 1.2MB | 1.2MB | -93.0B |
| `enterpriseSearch` | 956.4KB | 956.4KB | -36.0B |
| `esql` | 835.6KB | 835.6KB | -56.0B |
| `esqlDataGrid` | 153.5KB | 153.4KB | -95.0B |
| `eventAnnotationListing` | 209.4KB | 209.5KB | +46.0B |
| `filesManagement` | 104.4KB | 104.5KB | +46.0B |
| `fleet` | 2.7MB | 2.7MB | -319.0B |
| `graph` | 374.6KB | 374.6KB | +46.0B |
| `indexLifecycleManagement` | 153.5KB | 153.4KB | -15.0B |
| `indexManagement` | 792.3KB | 792.2KB | -62.0B |
| `infra` | 1.5MB | 1.5MB | -17.0B |
| `ingestPipelines` | 362.9KB | 362.9KB | +25.0B |
| `inspectComponent` | 46.9KB | 46.9KB | -27.0B |
| `kubernetesSecurity` | 225.6KB | 228.3KB | +2.6KB |
| `lens` | 2.0MB | 2.0MB | -458.0B |
| `links` | 103.4KB | 103.4KB | -18.0B |
| `maps` | 3.2MB | 3.2MB | +7.0KB |
| `ml` | 5.6MB | 5.6MB | +2.7KB |
| `monitoring` | 635.1KB | 635.1KB | -46.0B |
| `observability` | 2.0MB | 2.0MB | -114.0B |
| `observabilityAIAssistant` | 20.4KB | 20.3KB | -9.0B |
| `observabilityAIAssistantApp` | 658.2KB | 658.1KB | -57.0B |
| `observabilityAiAssistantManagement` | 106.5KB | 106.4KB | -48.0B |
| `observabilityShared` | 37.2KB | 37.3KB | +16.0B |
| `osquery` | 1.3MB | 1.3MB | +57.0B |
| `painlessLab` | 16.4KB | 16.4KB | -37.0B |
| `presentationUtil` | 41.1KB | 41.1KB | +56.0B |
| `profiling` | 363.9KB | 363.9KB | -15.0B |
| `savedObjectsManagement` | 80.7KB | 80.7KB | -15.0B |
| `searchAssistant` | 199.1KB | 199.0KB | -57.0B |
| `searchInferenceEndpoints` | 219.0KB | 219.0KB | +44.0B |
| `searchPlayground` | 243.4KB | 243.4KB | -11.0B |
| `searchQueryRules` | 131.3KB | 131.3KB | -9.0B |
| `security` | 548.8KB | 548.8KB | +17.0B |
| `securitySolution` | 12.0MB | 12.0MB | -1.3KB |
| `sessionView` | 357.1KB | 357.1KB | -9.0B |
| `slo` | 1.2MB | 1.2MB | -321.0B |
| `spaces` | 235.8KB | 235.8KB | -10.0B |
| `stackAlerts` | 88.9KB | 88.9KB | -33.0B |
| `stackConnectors` | 1.8MB | 1.8MB | -15.0B |
| `streamsApp` | 2.0MB | 2.0MB | -73.0B |
| `synthetics` | 1.1MB | 1.1MB | -133.0B |
| `transform` | 630.1KB | 630.3KB | +223.0B |
| `triggersActionsUi` | 1.8MB | 1.8MB | -72.0B |
| `unifiedDocViewer` | 672.8KB | 672.7KB | -160.0B |
| `unifiedSearch` | 310.9KB | 310.9KB | +23.0B |
| `uptime` | 492.0KB | 492.0KB | -72.0B |
| `urlDrilldown` | 109.3KB | 109.3KB | +16.0B |
| `visTypeTimeseries` | 440.7KB | 440.7KB | -15.0B |
| `visTypeVega` | 2.0MB | 2.0MB | +2.6KB |
| `visualizationListing` | 97.4KB | 97.4KB | +46.0B |
| `visualizations` | 334.7KB | 334.8KB | +46.0B |
| `workflowsManagement` | 2.3MB | 2.3MB | -36.0B |
| `workplaceAIApp` | 512.4KB | 512.4KB | +28.0B |
| total |  |  | +16.1KB |

#### Page load bundle
> Size of the bundles that are downloaded on every page load. Target size is below 100kb


| id | [before](https://github.com/elastic/kibana/commit/9a0ff9aa5714a7f550f8abf4b3a40f8d3c9ccf19) | [after](https://github.com/elastic/kibana/commit/1f05bb252104606cd608d35fa366b2d78a7c070d) | diff |
| --- | --- | --- | --- |
| `cloudLinks` | 34.2KB | 34.2KB | -9.0B |
| `core` | 521.8KB | 521.8KB | +38.0B |
| `elasticAssistant` | 266.2KB | 266.2KB | -59.0B |
| `kbnUiSharedDeps-npmDll` | 7.3MB | 7.4MB | +39.3KB |
| `kql` | 13.8KB | 13.8KB | +14.0B |
| `observabilityShared` | 66.6KB | 66.6KB | -28.0B |
| `savedObjectsTagging` | 20.9KB | 20.8KB | -26.0B |
| `security` | 71.7KB | 71.7KB | -9.0B |
| `sessionView` | 43.0KB | 43.0KB | -9.0B |
| `share` | 56.9KB | 56.9KB | -6.0B |
| `triggersActionsUi` | 114.6KB | 114.5KB | -19.0B |
| total |  |  | +39.2KB |

<details><summary>Unknown metric groups</summary>

#### ESLint disabled line counts
| id | [before](https://github.com/elastic/kibana/commit/9a0ff9aa5714a7f550f8abf4b3a40f8d3c9ccf19) | [after](https://github.com/elastic/kibana/commit/1f05bb252104606cd608d35fa366b2d78a7c070d) | diff |
| --- | --- | --- | --- |
| `stackAlerts` | 22 | 23 | +1 |

#### Total ESLint disabled count
| id | [before](https://github.com/elastic/kibana/commit/9a0ff9aa5714a7f550f8abf4b3a40f8d3c9ccf19) | [after](https://github.com/elastic/kibana/commit/1f05bb252104606cd608d35fa366b2d78a7c070d) | diff |
| --- | --- | --- | --- |
| `stackAlerts` | 22 | 23 | +1 |


</details>


### History
* :yellow_heart: [Build #436075](https://buildkite.com/elastic/kibana-pull-request/builds/436075) was flaky 60806097f0958833afdafa99d4227ba3bbed27c4
* :broken_heart: [Build #435863](https://buildkite.com/elastic/kibana-pull-request/builds/435863) failed f9c89c03c3d3ae7a2a7d2e19ac223d155f84838d
* :broken_heart: [Build #435476](https://buildkite.com/elastic/kibana-pull-request/builds/435476) failed 3c022875a7bff7fd17332c2278170cbaeb2fc4ff

cc @mgadewoll

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/436121","number":436121,"commit":"1f05bb252104606cd608d35fa366b2d78a7c070d"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/436075","number":436075,"commit":"60806097f0958833afdafa99d4227ba3bbed27c4"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/435863","number":435863,"commit":"f9c89c03c3d3ae7a2a7d2e19ac223d155f84838d"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/435476","number":435476,"commit":"3c022875a7bff7fd17332c2278170cbaeb2fc4ff"}],"number":436121}
buildkite-pr-comment-->

mgadewoll committed:
Merge remote-tracking branch 'upstream' into eui/selection-list-ui-updates

mgadewoll committed:
chore: update eui package

- includes EuiComboBox popover noSuggestions fix

mgadewoll committed:
refactor: update EuiSuperSelectItem usages to use prepend instead of icon

mgadewoll committed:
refactor: cleanup removed props after update with main

mgadewoll committed:
refactor: update dashboard selection components

- ensures the drilldown popover has an outer padding

- ensures the export app menu has a decent min width

mgadewoll committed:
chore: update i18n

mgadewoll committed:
test: update tests and snapshots

mgadewoll committed:
refactor: remove padding on SLO popovers

- the inner context menu already has a padding

mgadewoll committed:
refactor: add padding on unit_field popover

mgadewoll committed:
chore: update eui package

mgadewoll committed:
Merge remote-tracking branch 'upstream' into eui/selection-list-ui-updates

mgadewoll committed:
chore: cleanup eui test packages

mgadewoll committed:
chore: update eui packages

mgadewoll committed:
Merge remote-tracking branch 'upstream' into eui/selection-list-ui-updates

mgadewoll committed:
test: adjust locator selector after merge

mgadewoll committed:
Merge remote-tracking branch 'upstream' into eui/selection-list-ui-updates

mgadewoll committed:
test: revert commented out tests

mgadewoll committed:
refactor: update script_library_filter selectable

mgadewoll committed:
refactor: cleanup obsolete size prop on AttacksGroupTakeActionItemsProps

mgadewoll committed:
chore: update eui package

- includes additional ContextMenuPanel rerender change

mgadewoll committed:
do not cherry-pick: update eui testing package

mgadewoll committed:
test: update baseline snapshots

mgadewoll committed:
do not cherry-pick: update eui testing package

mgadewoll committed:
Merge remote-tracking branch 'upstream' into eui/pre-release-testing
